### PR TITLE
[faas] add `id` field

### PR DESCRIPTION
**Pull request by stuartnelson3**
## Motivation/summary

`id` identifies one or a grouping of functions as
a service.

## Checklist

- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/master/CHANGELOG.asciidoc)
- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/master/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)
- [ ] Documentation has been updated

## How to test these changes

send some events with `faas.id` to the apm-server. verify that the indexed documents have `faas.id`.

### CODE REVIEW

File: model/modeldecoder/v2/model.go [hunk 1]
```diff
@@ -97,6 +97,8 @@ type context struct {
 }
 
 type faas struct {
+	// A unique identifier representing one or several functions.
```
Comment (AlexanderWert) [inline model/modeldecoder/v2/model.go:100]:
```suggestion
	// A unique identifier of the invoked serverless function.
```

File: docs/spec/v2/transaction.json [hunk 2]
```diff
@@ -826,6 +826,13 @@
             "string"
           ]
         },
+        "id": {
+          "description": "A unique identifier representing one or several functions.",
```
Comment (AlexanderWert) [inline docs/spec/v2/transaction.json:830]:
```suggestion
          "description": "A unique identifier of the serverless function the transaction belongs to.",
```

**Review by AlexanderWert — APPROVED** [review]
LGTM, only a small proposal to change the field description

**Review by marclop — COMMENTED** [review]
change looks good, but shouldn't we add explicit Elasticsearch mappings for this new `faas.id`?

**Comment by marclop** [comment]
Verified with 42889b423.

**Pull request merged**